### PR TITLE
Update sweep gradient per #250

### PR DESCRIPTION
**Pull request by PeterConstable**
Partial fix for #250

### CODE REVIEW

File: OFF_AMD2_WD.md [hunk 1]
```diff
@@ -293,8 +293,7 @@ repetitions, use the result as the maximum stop offset for specified color
 stops, and set the extend mode to *reflect*.
 
 NOTE: Sweep gradients have special considerations in relation to color line
-extend modes and do not support repetition of gradient patterns. See
-5.7.11.1.2.4 for details.
+extend modes. See 5.7.11.1.2.4 for details.
```
Comment (drott) [inline OFF_AMD2_WD.md:296]: Entirely optional: Perhaps "Special considerations apply to color line extend modes in sweep gradients. See […]" reads a little more smoothly.

**Review by drott — APPROVED** [review]
Thanks for the quick fix.

File: OFF_AMD2_WD.md [hunk 2]
```diff
@@ -555,14 +554,17 @@ gradient, or as an “angular” gradient.
...
-counter-clockwise direction; for example, if the start and end angles are both
-0°, then stop offset 0.1 is at 36° counter-clockwise from the direction of the
-y-axis.
+offset 1 aligned with the ending angle. The color line progresses from the start
+angle to the end angle in the counter-clockwise direction; for example, if the
+start and end angles are both 0°, then stop offset 0.1 is at 36°
+counter-clockwise from the direction of the positive x-axis. For each position
+along the circular arc, from start to end in the counter-clockwise direction, a
+ray from the center outward is painted with the color of the color line at the
+point where the ray passes through the arc.
```
Comment (drott) [inline OFF_AMD2_WD.md:567]: Works for me.

File: OFF_AMD2_WD.md [hunk 3]
```diff
@@ -579,9 +581,9 @@ Because a sweep gradient is defined using start and end angles, the gradient
 does not need to cover a full 360° sweep around the center. This is illustrated
 in figure 5.30:
 
-![A sweep gradient, from red to yellow, with start angle of -60° and an end angle of 60°.](images/colr_conic_gradient_start_stop_angles.png)
+![A sweep gradient, from red to yellow, with start angle of 30° and an end angle of 150°.](images/colr_conic_gradient_start_stop_angles.png)
 
-**Figure 5.30 A sweep gradient, from red to yellow, with start angle of -60° and an end angle of 60°.**
+**Figure 5.30 A sweep gradient, from red to yellow, with start angle of 30° and an end angle of 150°.**
```
Comment (drott) [inline OFF_AMD2_WD.md:586]: I believe the changed angles are due to the y->x axis change, right? I think it'd still be useful to cover the 0-transition, or in other words go from some -Θ to +Θ.

**Comment by PeterConstable** [comment]
@drott I added more changes for the remainder of your comments in #250. Please re-review.

**Comment by drott** [comment]
> I believe the changed angles are due to the y->x axis change, right? I think it'd still be useful to cover the 0-transition, or in other words go from some -Θ to +Θ.

PS: That could be a follow-up CL to update the example.

**Pull request merged**